manim/mobject/svg/text_mobject.py

```diff
@@ -337,122 +316,213 @@ def text2svg(self):
+            font = setting.font
+            slant = self.str2slant(setting.slant)
+            weight = self.str2weight(setting.weight)
+            text = self.text[setting.start : setting.end].replace("\n", " ")
+            context.select_font_face(font, slant, weight)
+            if setting.line_num != last_line_num:
+                offset_x = 0
+                last_line_num = setting.line_num
+            tempx = START_X + offset_x
+            tempy = START_Y + line_spacing * setting.line_num
+            char_offset_x = 0
+            char_height = tempy - size / 2 - (line_spacing - size)
+            # context.set_font_matrix(cairo.Matrix(13, 0, 0, 13, 0, 0))
+            # print(context.get_font_matrix(),size)
+            # if (context.text_extents("99").width-context.text_extents("11").width)/size > 0.13:
+            # for char_index in range(text.__len__()):
+
+            # else:
+
+            print(context.text_extents("  ").width)
+            """
+            for char_index in range(text.__len__()):
+                (x, y, width, height, dx, dy) = context.text_extents("a" + text[char_index] + "a")
+                (_, _, widtha, _, _, _) = context.text_extents("a")
+                width -= 2 * widtha
+                context.rectangle(tempx + char_offset_x, char_height, width, size)
+                context.fill()
+                char_offset_x += width
+            """
+            context.move_to(tempx, tempy)
+            context.show_text(text)
+            offset_x += context.text_extents(text)[4]
+        surface.finish()
         return file_name
 
 
-class TextWithFixHeight(Text):
-    def __init__(self, text, **kwargs):
-        Text.__init__(self, text, **kwargs)
-        max_height = Text("(gyt{[/QW", **kwargs).get_height()
-        rectangle = Rectangle(
-            width=0, height=max_height, fill_opacity=0, stroke_opacity=0, stroke_width=0
-        )
-        self.submobjects.append(rectangle)
+"""
+paragraph paragraph.chars is VGroup() of each lines and each line is VGroup() of that line's characters 
+that mean you can use it like 
+    paragraph[0:5] or paragraph.chars[0:5] to access first five lines
+    paragraph[0][0:5] or paragraph.chars[0][0:5] to access first line's first five characters
+paragraph or paragraph[] or paragraph.chars[][] will create problems when using Transform() because of invisible characters 
+so, before using Transform() remove invisible characters by using remove_invisible_chars()
+for example self.play(Transform(remove_invisible_chars(paragraph.chars[0:2]), remove_invisible_chars(paragraph.chars[3][0:3])))
+paragraph(" a b", " bcd\nefg") is same as paragraph(" a b", " bcd", "efg")
+that means paragraph[2] is "efg"
+"""
 
 
 class Paragraph(VGroup):
     CONFIG = {
-        "line_spacing": 0.1,
-        "alignment": "center",
+        "line_spacing": -1,
+        "alignment": None,
     }
 
     def __init__(self, *text, **config):
         Container.__init__(self, **config)
```

Comment:
Why does this run `Container.__init__` if it inherits from VGroup?

Reply:
I think i did it just to `digest_config(self, config)`
Let me try this in that way.

Reply:
i needed init because i thought i need to init according to our need you can check. I mean custom init.

Reply:
Yes, but why are you running _`Container`s_ `__init__` method? Shouldn't you run `VGroup.__init__` since `Paragraph` inherits from `VGroup`?